specification/web/resource-manager/Microsoft.Web/stable/2021-02-01/WorkerApps.json

```diff
@@ -0,0 +1,443 @@
+      "get": {
+        "tags": [
+          "WorkerApps"
+        ],
+        "summary": "Get the properties of a Worker App.",
+        "description": "Description for Get the properties of a Worker App.",
+        "operationId": "WorkerApps_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the Worker App.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/WorkerApp"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../2021-02-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Worker App": {
+            "$ref": "./examples/GetWorkerApp.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "WorkerApps"
+        ],
+        "summary": "Create or update a Worker App.",
+        "description": "Description for Create or update a Worker App.",
+        "operationId": "WorkerApps_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the Worker App.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "workerAppEnvelope",
+            "in": "body",
+            "description": "Configuration details of the Worker App",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/WorkerApp"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "No change performed on the Worker App.",
+            "schema": {
+              "$ref": "#/definitions/WorkerApp"
+            }
+          },
+          "201": {
+            "description": "Worker App create or update has been started.",
+            "schema": {
+              "$ref": "#/definitions/WorkerApp"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../2021-02-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
```

Comment:
>2021-02-01 [](http://example.com/codeflow?start=26&length=10)

This isn't blocking, but if the common definitions are versioned, then shouldn't they be in the version's directory?

Reply:
They actually are in the version's directory, but since we generate the swagger, the path is a relative path from the RP root path.